scipy/stats/_axis_nan_policy.py

```diff
@@ -43,8 +55,16 @@ def _broadcast_shapes(shapes, axis=None):
 
     # Remove the shape elements of the axes to be ignored, but remember them.
     if axis is not None:
-        axis = np.atleast_1d(axis)
         axis[axis < 0] = n_dims + axis[axis < 0]
+        axis = np.sort(axis)
```

Comment:
The was actually needed to fix a bug in the existing `_broadcast_shapes`, which was supposed to work for axis tuples already - but previously it only worked if the axes were already sorted.